nodejs/week1/warmup.js

```diff
@@ -0,0 +1,18 @@
+class Circle{
+    constructor(radius){
+        this.radius = radius;
+    }
+
+    getDiameter= () => this.radius * 2;
+    getCircumference = () => 2 * Math.PI * this.radius;
+    getArea = () =>  Math.PI *(this.radius * this.radius);
```

Comment:
Using the short-hand arrow syntax only seems to work from version 12 of Node. This is not a problem, just something to be aware of.

Reply:
noted, thanks